pkg/apis/v1/labels.go

```diff
@@ -51,6 +51,7 @@ const (
 	KubeletCompatibilityAnnotationKey          = apis.CompatibilityGroup + "/v1beta1-kubelet-conversion"
 	NodeClassReferenceAnnotationKey            = apis.CompatibilityGroup + "/v1beta1-nodeclass-reference"
 	NodeClaimTerminationTimestampAnnotationKey = apis.Group + "/nodeclaim-termination-timestamp"
+	StorageVersion                             = apis.Group + "/storage-version"
```

Comment:
```suggestion
	StorageMigrated                             = apis.Group + "/storage-migrated"
```

Should this just be a true/false to signal that we have migrated to the new storage version?

Reply:
storage-version-migrated?